### PR TITLE
docs: update readme with mod panel instructions

### LINKED ISSUE #698: DOC: Add documentation on how to run moderation in development
# Problem
Currently we are missing instructions on how we run moderation in development.

### Requirements

- [ ] Add clear steps on how to use the moderation panel in development using our local db
- [ ] Add clear instructions on how to work with auth in development

# Implementation notes
[Suggest how this might be implemented.]

### PR DESCRIPTION
**Pull request by NabbeunNabi**
Resolves #698 

## 🔧 What changed
Before there were no clear instructions on how to work in the moderation panel on dev. Due to the data being protected behind auth. There are now instructions that link those to the backend `README` so that they can set up the backend and then instructions how to run the backend and frontend so they connect. How to log in and then see the changes in the mod panel on the frontend.

### CODE REVIEW

**Comment by theyokohamalife** [comment]
LGTM but @ProgressChris88 and @ShizuokaTerrier are also taking a look at it and following the steps to confirm that they work.

**Comment by ShizuokaTerrier** [comment]
Followed the steps and successfully logged into the mod dashboard in dev. 

Note: For anyone using **nvm** for the first time, it might take a long time to install and look like a clip from the matrix while doing so.

**Comment by ProgressChris88** [comment]
Followed the steps and had no issues. There could possibly be confusion when logging into Firebase CLI for the first time through their error report system. Maybe we could just mention that logging into Fireblase CLI for the first time will prompt you to login and you can login through your Google account for example.

File: README.md [hunk 1]
```diff
@@ -91,16 +93,44 @@ $ yarn prepare
 $ yarn dev
 ```
 
-## Testing a production build
+## Running development (moderation panel with our local db)
```
Comment (ermish) [inline README.md:96]: I think we should make these optional sections accordion menus
- mod panel
- running production mode to test a build

**Comment by NabbeunNabi** [comment]
> Followed the steps and had no issues. There could possibly be confusion when logging into Firebase CLI for the first time through their error report system. Maybe we could just mention that logging into Fireblase CLI for the first time will prompt you to login and you can login through your Google account for example.

@ProgressChris88 I will make a ticket for this on the backend README as I think it was part of that one you were linked to.

File: README.md [hunk 2]
```diff
@@ -91,16 +93,51 @@ $ yarn prepare
...
+
+```bash
+# install dependencies
+$ npm install
 
+# start the local db
+$ npm run dev:startlocaldb
+
+# For Development, start a dev server backend
+$ yarn dev
```
Comment (Anissa3005) [inline README.md:111]: The backend doesn't use `yarn` so they have to run `npm run dev` to start the server. Probably just a spelling mistake 😅

Comment (NabbeunNabi) [reply README.md:111]: Ahh it doesn't but in my machine it doesn't care which I use so I was definitely just thinking of what I type 😅

**Review by theyokohamalife — APPROVED** [review]
Thank you for keeping our docs updated!

**Comment by theyokohamalife** [comment]
This will need approval from @ermish to be merged since he requested changes.

**Review by ermish — APPROVED** [review]
readme level up!

**Pull request merged**